src/test/components/UrlManager.test.js

```diff
@@ -328,4 +329,47 @@ describe('UrlManager', function () {
     // timeline track organization state.
     expect(getTimelineTrackOrganization(getState()).type).toBe('active-tab');
   });
+
+  it('can handle a from-post-message data source', async () => {
+    const { getState, waitUntilUrlSetupPhase, createUrlManager } = setup(
+      '/from-post-message/'
+    );
+    const log = console.log;
+    (console: any).log = jest.fn();
```

Comment:
It would be better to use `jest.spyOn` so that the clean up is automatic:
```suggestion
    jest.spyOn(console, 'log').mockImplementation(() => {});
```
and remove the clean-up at the end of the testcase.

Reply:
It's been awhile since I've been in these tests, thanks that's cleaner.